specifyweb/frontend/js_src/lib/components/QueryBuilder/Fields.tsx

```diff
@@ -163,18 +162,27 @@ export function QueryFields({
     }, [fields.length])
   );
 
+  const isBasic = React.useContext(IsQueryBasicContext);
+
   return (
-    <Ul className="flex-1 overflow-y-auto" forwardRef={fieldsContainerRef}>
+    <Ul
+      className={`flex-1 items-center overflow-y-auto
```

Comment:
Could you please indent this properly? A new line after first \` and before last \`